### PR TITLE
Add build instructions

### PR DESCRIPTION
**Pull request by sgladkov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,12 @@
...
+
+### Build the project
+```
+> git clone https://github.com/LATOKEN/lachain
+> cd lachain
+> git submodule update --init --recursive
+> dotnet build -c Debug
+> dotnet publish -p:Configuration=Debug --force -p:PublishReadyToRun=true -p:IncludeAllContentForSelfExtract=true --self-contained true -p:PublishSingleFile=true --runtime linux-x64 src/Lachain.Console/Lachain.Console.csproj
+```
+After that place resulting `Lachain.Console` file in the same folder with appropriate config.json and start it
```
Comment (dushyantb) [inline README.md:12]: we shall add the path where the executable is generated. i.e. `/src/Lachain.Console/bin/Debug/net5.0/linux-x64/publish/Lachain.Console`

**Review by HMAshiqulIslam — COMMENTED** [review]
Should we mention - this is specific to linux machine ?

File: README.md (hunk 1 above)
Comment (sgladkov) [reply README.md]: Done

**Comment by sgladkov** [comment]
> There is another line before this line in the file I use to build: dotnet restore

`restore` is called during the `publish`,  I don't sure we need explicit call here

> I have it as 'dotnet build -c Release' instead of 'dotnet build -c Debug'

Fixed

> Should we mention - this is specific to linux machine ?

I didn't tried to build it on other system. Can you describe the key differences? @HMAshiqulIslam

**Comment by HMAshiqulIslam** [comment]
for windows machine: 

dotnet publish -p:Configuration=Debug --force -p:PublishReadyToRun=true -p:IncludeAllContentForSelfExtract=true --self-contained true -p:PublishSingleFile=true --runtime win-x64 src/Lachain.Console/Lachain.Console.csproj

Also generate filepath also changes to: 
/src/Lachain.Console/bin/Release/net5.0/win-x64/publish/

I am not sure about the other os.

**Comment by dushyantb** [comment]
BTW this above command is not working on `OSX` and getting following error:

`error NETSDK1094: Unable to optimize assemblies for performance: a valid runtime package was not found. Either set the PublishReadyToRun property to false, or use a supported runtime identifier when publishing. [/src/Lachain.Console/Lachain.Console.csproj]`

command I'm using on `OSX` is 

`dotnet publish -p:Configuration=Debug -p:PublishTrimmed=true -p:SelfContained=true -p:PublishSingleFile=true -p:TrimUnusedDependencies=true -p:IncludeNativeLibrariesForSelfExtract=true -p:IncludeAllContentForSelfExtract=true --runtime osx-x64 src/Lachain.Console/Lachain.Console.csproj`

**Review by HMAshiqulIslam — APPROVED** [review]

**Review by tbssajal — APPROVED** [review]

**Pull request merged**